examples/wearable/UIComponents/contents/user-inputs-elements/radio/radio.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>
+        Wearable UI
+    </title>
+    <link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+    <link href="../../../lib/tau/wearable/theme/default/tau.circle.css" />
+    <link href="../../../css/style.css" rel="stylesheet" />
+    <link href="../../../css/style.circle.css" />
+    <script src="../../../js/circle-helper.js"></script>
+    <script src="../../../lib/tau/wearable/js/tau.js"></script>
+</head>
+
+<body>
+    <div class="ui-page">
+        <header class="ui-header">
+            <h2 class="ui-title">Radio</h2>
+        </header>
+        <div class="ui-content">
+            <ul class="ui-listview">
+                <radiogroup>
```

Comment:
I have a question.
What's the "radiogroup"?
Is it needed for sample?

Reply:
Thank you for your advice. I removed this tag. It wasn't necessary.